diff --git a/src/tier0/dbg.cpp b/src/tier0/dbg.cpp
index f794c1e..89d004a 100644
--- a/src/tier0/dbg.cpp
+++ b/src/tier0/dbg.cpp
@@ -90,7 +90,7 @@ bool Plat_IsInDebugSession()
 	if (sysctl(mib, 4, &info, &size, NULL, 0) == -1)
 	    return false;
 	return ((info.p_psflags & PS_TRACED) != 0);
-#elif IsLinux()
+#elif IsLinux() || IsAndroid()
 	static FILE *fp;
 	if ( !fp )
 	{
diff --git a/src/tier1/netadr.cpp b/src/tier1/netadr.cpp
index e4b64d6..1434ad5 100644
--- a/src/tier1/netadr.cpp
+++ b/src/tier1/netadr.cpp
@@ -645,7 +645,7 @@ size_t CIPAndPort::ToSockadr(void *addr, size_t addr_size) const
 			}
 			auto *s = (struct sockaddr_in*)addr;
 			s->sin_family = AF_INET;
-			s->sin_addr.s_addr = BigDWord( INADDR_LOOPBACK );
+			s->sin_addr.s_addr = BigDWord( (uint32)INADDR_LOOPBACK );
 			s->sin_port = BigWord( m_usPort );
 			struct_size = sizeof(sockaddr_in);
 		}
@@ -660,7 +660,7 @@ size_t CIPAndPort::ToSockadr(void *addr, size_t addr_size) const
 			}
 			auto *s = (struct sockaddr_in*)addr;
 			s->sin_family = AF_INET;
-			s->sin_addr.s_addr = BigDWord( INADDR_BROADCAST );
+			s->sin_addr.s_addr = BigDWord( (uint32)INADDR_BROADCAST );
 			s->sin_port = BigWord( m_usPort );
 			struct_size = sizeof(sockaddr_in);
 		}
